test/functional/mempool_accept.py

```diff
@@ -333,6 +340,34 @@ def run_test(self):
             maxfeerate=0,
         )
 
+        # Prep for tin-tx tests with wsh(OP_TRUE) output
+        seed_tx = self.wallet.send_to(from_node=node, scriptPubKey=script_to_p2wsh_script(CScript([OP_TRUE])), amount=COIN)
+        self.generate(node, 1)
+
+        self.log.info('A tiny transaction(in non-witness bytes) that is disallowed')
+        tx = CTransaction()
+        tx.vin.append(CTxIn(COutPoint(int(seed_tx[0], 16), seed_tx[1]), b"", SEQUENCE_FINAL))
+        tx.wit.vtxinwit = [CTxInWitness()]
+        tx.wit.vtxinwit[0].scriptWitness.stack = [CScript([OP_TRUE])]
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN] + ([OP_0] * (MIN_PADDING - 2)))))
+        # Note it's only non-witness size that matters!
+        assert_equal(len(tx.serialize_without_witness()), 64)
```

Comment:
nit: based on the comment I think something like this would be appropriate to add: (import `assert_greater_than` from util)
```suggestion
        # Note it's only non-witness size that matters!
        assert_greater_than(len(tx.serialize()), 64)
        assert_equal(len(tx.serialize_without_witness()), 64)
```

Reply:
done